HopAddition.elm

```diff
@@ -35,11 +35,18 @@ initialHopAddition =
   , boilTime = { value = 60, timeUnit = Minute }
   }
 
-type Msg = SetAmount String | SetBoilTimeValue String | SetVariety String
+type Msg =
+    Delete
+  | SetAmount String
+  | SetBoilTimeValue String
+  | SetVariety String
 
 update : Msg -> Model -> Model
 update msg model =
   case msg of
+    Delete ->
+      model
```

Comment:
Returning an unchanged model seems to confirm my suspicion that what I'm doing is an anti-pattern